saleor/graphql/app/tests/mutations/test_app_activate.py

```diff
@@ -100,24 +120,27 @@ def test_app_has_more_permission_than_user_requestor(
     variables = {
         "id": id,
     }
+
+    # when
     response = staff_api_client.post_graphql(
         query, variables=variables, permissions=(permission_manage_apps,)
     )
+
+    # then
     content = get_graphql_content(response)
     app_data = content["data"]["appActivate"]["app"]
     app_errors = content["data"]["appActivate"]["appErrors"]
     app.refresh_from_db()
 
-    assert not app_data
-    assert len(app_errors) == 1
-    assert app_errors[0]["field"] == "id"
-    assert app_errors[0]["code"] == AppErrorCode.OUT_OF_SCOPE_APP.name
-    assert not app.is_active
+    assert not app_errors
```

Comment:
Don't you think that we should change the name of the test? To be something more meaningful? which will emphasize that users with permission manage_app can activate/deactivate the app no matter what perms are assigned to the app.

Reply:
Right now it's described the situation that is testing, so maybe adding a short description at the begging of the test will help?